testimony/__init__.py

```diff
@@ -444,6 +448,93 @@ def validate_docstring_report(testcases):
         return -1
 
 
+def validate_values_report(testcases):
+    """Check that given token contains one of allowed values."""
+    result = {}
+    invalid_docstring_count = 0
+    invalid_token_value_count = 0
+    missing_docstring_count = 0
+    missing_token_count = 0
```

Comment:
This is not being used to report later on. I think we can just drop it. Also drop line 472.

Reply:
Fixed in #142, as `validate-values` command was dropped.